docs/Installation.md

```diff
@@ -127,32 +123,65 @@ sudo systemctl reload apache2
 
 For all commands below become root:
 
-``su -l``
+```sh
+su -l
+```
+
+#### Update list of package repositories
+Create the file `/usr/local/etc/pkg/repos/FreeBSD.conf` with the following
+content, unless it is already updated:
+
+```
+FreeBSD: {
+url: "pkg+http://pkg.FreeBSD.org/${ABI}/latest",
+}
+```
```

Comment:
What would happen if we removed this? (From here and from Engine as well).

Reply:
Older packages would be installed. I added this when there was an inconsistency between the packages in "monthly", which is the default.